tracing-subscriber/src/registry/fmt.rs

```diff
@@ -315,9 +311,9 @@ where
     // TODO(david): consider an alternative location for this.
```

Comment:
?

Reply:
I think it's in the right spot? Do you have a take/vision as to where this should be placed other than `FmtContext`?